MiddleWidgets/SettingWidget/SuSoftware.cpp

```diff
@@ -33,33 +34,80 @@ QWidget *SuSoftware::getUnitWidget(QWidget *parent)
     QLabel* label2 = new QLabel(SettingUnitContainer);
     QLabel* label3 = new QLabel(SettingUnitContainer);
     QLabel* label4 = new QLabel(SettingUnitContainer);
+    QLabel* label5 = new QLabel(SettingUnitContainer);
+    QLabel* label6 = new QLabel(SettingUnitContainer);
 
     label1->setObjectName("labelSettingHighColor");
     label2->setObjectName("labelSettingHighColor");
     label3->setObjectName("labelSettingHighColor");
     label4->setObjectName("labelSettingHighColor");
-
-    label1->setText(tr("本软件 名称：BesLyric ") + " ( "+VERSION_NAME + " )");
-
-    label2->setText(tr("<p style='line-height:130%'>本软件 试图提供给用户一个 上手简单、功能实用的 滚动歌词制作软件，为广大网易云音乐爱好者提供一个良好的歌词制作体验</p>"));
-
-    label3->setText(tr("本软件 详细使用说明 以及 解决常见问题 : <a style='color:#666666;' href='https://www.cnblogs.com/BensonLaur/p/9695769.html'>点击这里</a>"));
-
-    label4->setText(tr("<p style='line-height:130%'>本软件 <a style='color:#666666;' href='https://blog.csdn.net/testcs_dn/article/details/37722355'>自由开源</a>"
-    " ，License 为 GPL，使用跨平台框架 C++ 跨平台框架 QT 实现，对软件实现感兴趣的童鞋可以在此查看所有实现代码："
-    "<a style='color:#666666;' href='https://github.com/BensonLaur/Beslyric-for-X'>GitHub 源码</a></p>"));
-
+    label5->setObjectName("labelSettingHighColor");
+    label6->setObjectName("labelSettingHighColor");
+
+    label1->setText(tr("<p style='line-height:130%'>本软件名为 BesLyric。BesLyric 是一款 <a style='color:#666666;' href='https://blog.csdn.net/testcs_dn/article/details/37722355'>免费且开源</a> 的软件，我们开发本软件的目的，是为了提供给用户一个 上手简单、功能实用的 滚动歌词制作软件，为广大网易云音乐爱好者提供一个良好的歌词制作体验。</p>"));
+
+    label2->setText(tr("软件使用说明: <a style='color:#666666;' href='https://www.cnblogs.com/BensonLaur/p/9695769.html'>点击这里</a>"));
+
+    label3->setText(tr("<p style='line-height:130%'>软件用户交流: <a style='color:#666666;' href='https://jq.qq.com/?_wv=1027&k=5Uc1a79'>QQ群 1021317114</a></p>"));
+
+    QString OSName = QSysInfo::prettyProductName();
```

Comment:
As per the [doc](https://doc.qt.io/qt-5/qsysinfo.html#prettyProductName) of `QSysInfo::prettyProductName()` :

> The result of this function is suitable for displaying to the user, but not for long-term storage, as the string may change with updates to Qt.

Yes, it is. On Windows 10 10.0.19043 (21H1), it returns "Windows 10 Version 2009". For the sake of accuracy, I think we need to use the combination of `currentCpuArchitecture()` , `kernelType()` , `kernelVersion()` , `productType()` and `productVersion()` .